### PR TITLE
Add TensorFlow example predict fns

### PR DESCRIPTION
**Pull request by regadas**

### CODE REVIEW

File: zoltar-tensorflow/src/main/java/com/spotify/zoltar/tf/TensorFlowPredictFn.java [hunk 1]
```diff
@@ -33,4 +47,64 @@
...
 
+  /**
+   * TensorFlow Example prediction function.
+   *
+   * @param outTensorExtractor Function to extract the output value from a JTensor
+   * @param fetchOp            operation to fetch.
+   */
+  static <InputT, ValueT> TensorFlowPredictFn<InputT, Example, ValueT> example(
+      final Function<JTensor, ValueT> outTensorExtractor,
+      final String fetchOp) {
```
Comment (ravwojdyla) [inline zoltar-tensorflow/src/main/java/com/spotify/zoltar/tf/TensorFlowPredictFn.java]: Should the fetchOps be a `List<String>` (same in the method below)?

Comment (regadas) [reply zoltar-tensorflow/src/main/java/com/spotify/zoltar/tf/TensorFlowPredictFn.java]: Hi @ravwojdyla, yup we can do that! However we have the following; Multiple `fetchOps` yield different `JTensor`'s which will mean that `ValueT` might not be the same for every op. We would need to turn `ValueT` into `Map<String, Object>` where key is the op and value the result of a `Function<JTensor, Object>`.

I guess the signature of this method would look something like:

```java
  static <InputT> TensorFlowPredictFn<InputT, List<Example>, Map<String, Object>> exampleBatch(
      final Map<String, Function<JTensor, Object>> ops)
```

ppl will need to do some cast afterwards. WDYT?

Comment (ravwojdyla) [reply zoltar-tensorflow/src/main/java/com/spotify/zoltar/tf/TensorFlowPredictFn.java]: I wonder - could we expose a `Map[String, JTensor]` and let users fetch whatever type/value they want/know is in the Tensor?

Comment (regadas) [reply zoltar-tensorflow/src/main/java/com/spotify/zoltar/tf/TensorFlowPredictFn.java]: @ravwojdyla yup! I was exploring that right now 😄 

looking like this 
```java
  static <InputT, ValueT> TensorFlowPredictFn<InputT, List<Example>, ValueT> exampleBatch(
      final Function<Map<String, JTensor>, ValueT> outTensorExtractor,
      final String... fetchOps)
```

Comment (regadas) [reply zoltar-tensorflow/src/main/java/com/spotify/zoltar/tf/TensorFlowPredictFn.java]: @ravwojdyla updated! can you have another look?

Comment (ravwojdyla) [reply zoltar-tensorflow/src/main/java/com/spotify/zoltar/tf/TensorFlowPredictFn.java]: @regadas I like! What do you think?

Comment (regadas) [reply zoltar-tensorflow/src/main/java/com/spotify/zoltar/tf/TensorFlowPredictFn.java]: @ravwojdyla I think it's good!

Comment (ravwojdyla) [reply zoltar-tensorflow/src/main/java/com/spotify/zoltar/tf/TensorFlowPredictFn.java]: @regadas :shipit:

**Comment by ravwojdyla** [comment]
:shipit:

**Pull request merged**